tests/conftest.py

```diff
@@ -42,7 +42,7 @@ def __prepared_function(self, *args, **kwargs):
         if not kwargs:
             modifier, modifier_dict = 'call', {}
             fn_abi = [x for x in self._function.contract_abi if x['name'] == self._function.function_identifier].pop()
-            modifier_dict.update({'gas': fn_abi['gas'] * 1000})  # To make tests faster just supply some high gas value.
```

Comment:
How does this make tests faster? Genuinely curious

Reply:
If you supply a gas value, web3 doesn't run an estimateGas. estimateGas being very expensive as it does the the same call over and over to to estimate the gas.

Reply:
Oh, that makes complete sense! I wonder if we could just set it to the block gas limit since it automines and there's no reason for a transaction to cost more than 6 or 7 million gas (or whatever it is now)

Reply:
And then check for the actual execution amount after it runs